.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected behavior
+
+### Actual behavior
+
+Exceptionally long log output should be posted as a gist (https://gist.github.com).
+
+### Steps to reproduce the behavior
+
+Please include the following if applicable:
+
+- [ ] Manual steps to reproduce issue
+- [ ] The output of `l0 admin debug`
```

Comment:
We mention in the CONTRIBUTING.md to remove sensitive material from issues, but I can see them still slipping in here I guess. I can remove this bit for now or add clarification not to put in tokens